build-package.sh

```diff
@@ -18,7 +18,7 @@ else
 	export TERMUX_BUILD_PACKAGE_CALL_DEPTH=$((TERMUX_BUILD_PACKAGE_CALL_DEPTH+1))
 fi
 
-set -e -o pipefail -u
+set -e -E -o pipefail -u
```

Comment:
This isn't a very common set flag.
```yml
-E      If set, any trap on ERR is inherited by shell functions,
        command substitutions, and commands executed in a
        subshell environment.  The ERR trap is normally not
        inherited in such cases.
```
Consider adding a comment.

Reply:
Without that trap does not follow functions.

Reply:
Which would be good to mention.